tutorials/time-freq/plot_sensors_time_frequency.py

```diff
@@ -136,7 +136,13 @@
 # To this we'll use the function :func:`mne.time_frequency.tfr_morlet`
 # but you can also use :func:`mne.time_frequency.tfr_multitaper`
 # or :func:`mne.time_frequency.tfr_stockwell`.
-
+#
+# .. note::
+#       The ``decim`` parameter reduces the sampling rate of the time-frequency
+#       decomposition by the defined factor. This is usually done to reduce
+#       memory usage. For more information refer to the documentation of
+#       :func:`mne.time_frequency.tfr_morlet`
```

Comment:
```suggestion
#       :func:`mne.time_frequency.tfr_morlet`.
```

Nitpick but you might want to add a period.
Otherwise LGTM!

Reply:
Ahh, missed that one.